Maquetado/scss/navbar.scss

```diff
@@ -0,0 +1,27 @@
+.head {
+  align-items: center;
+  background-color: white;
+  box-shadow: 2px 2px 15px #ccc;
+  display: flex;
+  flex-flow: row nowrap;
+  height: $nav-height;
+  justify-content: space-between;
+  position: fixed;
+  width: 100%;
+  z-index: 1;
+  
+  .title-head {
+    color:$gray-dark;
+    font-size: $text-big;
+    padding: 0 15px;
+  }
+  
+  .logo {
+    padding: 0 10px;
+  }
+}
+
+.space {
```

Comment:
you should wrap the content inside a div and add padding to it, something like this:
```
<header>...</header>
<div class="content">
  ...
</div>
```